src/Particular.CodeRules/AwaitOrCaptureTasks/AwaitOrCaptureTasksAnalyzer.cs

```diff
@@ -23,6 +23,8 @@ public class AwaitOrCaptureTasksAnalyzer : DiagnosticAnalyzer
         /// <param name="context">The context.</param>
         public override void Initialize(AnalysisContext context)
         {
+            context.EnableConcurrentExecution();
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
```

Comment:
To address this new violation:

> error RS1025: Configure generated code analysis

Reply:
I wonder if `GeneratedCodeAnalysisFlags.None` would be better here. If there was a problem with generated code, how would we fix it? You'd have to modify the code generator first, and then bring in the new version into the project.

Seems like that might be too big of a blocker?

Reply:
But what if there is some generated code which violates our analyser? I think we'd want to know about it. Perhaps we can try it like this first and then relax the constraint later if it proves problematic?